src/metrics/policy_evaluations_latency.rs

```diff
@@ -0,0 +1,18 @@
+use crate::metrics::PolicyEvaluation;
+use lazy_static::lazy_static;
+use opentelemetry::{metrics::ValueRecorder, KeyValue};
+use std::time::Duration;
+
+lazy_static! {
+    static ref POLICY_EVALUATION_LATENCY: ValueRecorder<f64> =
+        opentelemetry::global::meter(super::METER_NAME)
+            .f64_value_recorder("kubewarden_policy_evaluation_latency_seconds")
+            .init();
+}
+
+pub fn record_policy_latency(latency: Duration, policy_evaluation: PolicyEvaluation) {
```

Comment:
Should we pass `policy_evaluation` by reference?

Reply:
I think it might have an impact on the `Into<Vec<KeyValue>>` impl, but we can try to do the impl for &PolicyEvaluation though.

@jvanz can you check?

Reply:
I don't want to over-optimize things, but given this code is called on each evaluation I was thinking about avoiding copies as much as possible